backend/pkg/api/data_access/user.go

```diff
@@ -6,19 +6,33 @@ import (
 	"math"
 	"time"
 
+	"github.com/gobitfly/beaconchain/pkg/api/services"
 	t "github.com/gobitfly/beaconchain/pkg/api/types"
 	"github.com/gobitfly/beaconchain/pkg/commons/utils"
 	"github.com/pkg/errors"
 )
 
 type UserRepository interface {
+	GetUserExists(email string) (bool, error)
+	AddUser(email, password string) error
 	GetUserCredentialInfo(email string) (*t.UserCredentialInfo, error)
 	GetUserIdByApiKey(apiKey string) (uint64, error)
 	GetUserInfo(id uint64) (*t.UserInfo, error)
 	GetUserDashboards(userId uint64) (*t.UserDashboardsData, error)
 	GetUserValidatorDashboardCount(userId uint64) (uint64, error)
 }
 
+func (d *DataAccessService) GetUserExists(email string) (bool, error) {
+	// TODO @patrick
+	return d.dummy.GetUserExists(email)
+}
+
+func (d *DataAccessService) AddUser(email, password string) error {
+	// TODO @patrick
```

Comment:
Minor: These probably shouldn't have a `@patrick` :)

Reply:
Anyone else instead, like `recy21` or `DATA-ACCESS`? Or just `TODO`? I'll just suggest `@DATA-ACCESS` for now

Reply:
both recy and data access are fine, or even a naked `// TODO` as we're creating a specific follow up bids anyways